integrationtests/tests/tests.rs

```diff
@@ -722,6 +726,34 @@ async fn set_lightning_invoice_expiry() -> Result<()> {
     task_group.shutdown_join_all().await
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn client_cannot_pay_expired_invoice() -> Result<()> {
+    let Fixtures {
+        fed,
+        user,
+        bitcoin,
+        lightning,
+        task_group,
+        ..
+    } = fixtures(2).await?;
+
+    fed.mine_and_mint(&user, &*bitcoin, sats(1100)).await;
+    fed.run_consensus_epochs(1).await;
+    let invoice = lightning.invoice(sats(1000), 1.into());
+    // This does not work correctly. It should sleep 2 seconds so that the invoice is expired when fund_outgoing_ln_contract is called
+    thread::sleep(Duration::from_secs(2));
+    fed.run_consensus_epochs(1).await;
```

Comment:
It will pass if you run two consensus epochs here, you need 1 for peers to send out the new time and another for that to be updated in consensus.  Perhaps you need to rearrange code in the LN module to do it in 1 epoch.

Reply:
Fixed hanging tests in https://github.com/fedimint/fedimint/pull/1074/commits/d1bf57e966283cd30cb3222fb307735dd9585c3e